scripts/build-desktop.sh

```diff
@@ -13,6 +13,16 @@ else
   ENV_FILE=".env"
 fi
 
+if [[ -n $GCP_GEOLOCATION_API_KEY ]]; then
+  if grep -q "^GCP_GEOLOCATION_API_KEY=" "$ENV_FILE"; then
```

Comment:
I usually use `-E` with grep (so `grep -qE`) when using regular expressions, though I tested this and it works as-is so NAB

Reply:
updated. it doesn't hurt to include the -E